.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
```

Comment:
this should be accompanied with a change in release repo:
```
build_root:
  from_repository: true
```

Reply:
Done on https://github.com/openshift/release/pull/25855